SharkCage/CageManager/CageLabeler.h

```diff
@@ -173,22 +235,63 @@ bool CageLabeler::ShowCageWindow()
 	return true;
 }
 
-void DisplayTokenInCageWindow(HWND *hwnd)
+static bool DisplayTokenInCageWindow(HWND *hwnd)
 {
 	std::wcout << L"starting display image" << std::endl;
+	
+	const std::string token_string(app_tokens.begin(), app_tokens.end());
+
+	std::string decoded_image = base64_decode(token_string);
+
+	DWORD image_size = decoded_image.length();
+
+	HGLOBAL h_memory = ::GlobalAlloc(GMEM_MOVEABLE, image_size);
```

Comment:
Shouldn't the memory be deallocated at some point with `GlobalFree`? Maybe use a smart pointer if it should but is needed beyond the scope of this method. 

Also, the [documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/aa366574(v=vs.85).aspx) mentions the following:
```text
The movable-memory flags GHND and GMEM_MOVABLE add unnecessary overhead and require locking to be used safely. They should be avoided unless documentation specifically states that they should be used.
```